scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -1012,11 +1052,46 @@ async def _apply_extension_setup_api_after_start(
+                    entry
+                    for entry in catalog_response.json().get("groups", [])
+                    if isinstance(entry, dict)
+                    and entry.get("group_id") == f"extension.{package_id}"
+                ),
+                None,
+            )
+            if group is None or group.get("revision") is None:
+                raise LiveQaError(
+                    f"Operator catalog does not declare group extension.{package_id} "
+                    "with a revision; cannot route non-secret setup values"
+                )
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)